gatsby-node.js

```diff
@@ -1,51 +1,52 @@
 /* eslint "no-console": "off" */
 
-const path = require("path");
-const _ = require("lodash");
-const moment = require("moment");
-const siteConfig = require("./data/SiteConfig");
+const path = require('path')
+const _ = require('lodash')
+const moment = require('moment')
+const siteConfig = require('./data/SiteConfig')
+
+const hasOwnProperty = (obj, property) => Object.prototype.hasOwnProperty.call(obj, property)
```

Comment:
I don't believe we actually need to use `Object.prototype.hasOwnProperty()` but to avoid any particular gotcha here left it in.